src/omotes_orchestrator/postgres_interface.py

```diff
@@ -115,12 +115,19 @@ def put_new_job(
         :param timeout_after: Maximum duration before the job is terminated due to timing out.
         """
         with session_scope() as session:
+            # If timeout_after is not given,
+            # timeout_after_ms is registered as null in the database.
+            if timeout_after is not None:
```

Comment:
I miss fmap.. :( Otherwise this would have been a oneliner hehe :P 

In case you are curious: https://hackage.haskell.org/package/base-4.20.0.1/docs/Prelude.html#v:fmap Allows you to perform an operation on an `Optional[whatever]` and it performs the operation unless the value is None, in which case the None value is preserved.

Reply:
Agree :D